src/Subscriptions/Broadcasters/LogBroadcaster.php

```diff
@@ -85,9 +85,7 @@ public function broadcast(Subscriber $subscriber, array $data)
      */
     public function broadcasts(?string $key = null): ?array
     {
-        return $key
-            ? Arr::get($this->broadcasts, $key)
-            : $this->broadcasts;
+        return Arr::get($this->broadcasts, $key);
```

Comment:
Are you sure that is the same?

Reply:
Yes: https://github.com/laravel/framework/blob/08e43fba6c4208acf04615c77037f186f88b8ba2/src/Illuminate/Support/Arr.php#L284-L286